R/count_teleconnections.R

```diff
@@ -93,6 +94,24 @@ count_watershed_teleconnections <- function(data_dir,
         # NOTE: CURRENTLY COUNTS NESTED WATERSHEDS; ADDITIONAL...
         # ... ALGORITHM NEEDED TO AVOID DOUBLE COUNTING
 
+        # TELECONNECTION - NUMBER OF CITIES USING SAME WATERSHEDS
+
+        # Load city data frame and join the list of city watersheds.
+        get_cities() -> city_df
+        as.data.frame(city_intake_ids) -> city_id
+        rename(city_id, DVSN_ID = city_intake_ids ) -> city_re
+        city_re$match <- city_re$DVSN_ID
```

Comment:
What was the need for matching? You have the list of DVSN ids in city_intake_ids, and you have the full city data table in city_df... Any reason why we can't simply filter city_df for the relevant DVSN ids and then count the resulting cities (as suggested above)?

Reply:
No reason, it's just the way my brain decided to work through the problem. What you did makes much more sense and I am trying to train myself to think more simply about these problems haha. Thank you! I'll make the changes and re-push.